http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/app/WdsAppInstallSpec.scala

```diff
@@ -23,12 +23,14 @@ class WdsAppInstallSpec extends BaseAppInstallSpec {
     mockAzureApplicationInsightsService
   )
 
-  it should "build wds override values" in {
+  it should "build wds override values for env" in {
```

Comment:
Is there any reasonable scenario for the `environmentBase="bee"` that would be worth testing similarly?

Reply:
at this point no - since the values are just strings and nothing is different. there used to be a test specific to a bee when config values were going to be formatted differently based on condition but that is now happening inside of terra-helm